packages/cookbook/src/accounts/access-keys/create-full-access-key.js

```diff
@@ -0,0 +1,41 @@
+const { KeyPair, keyStores, connect } = require('near-api-js');
+const path = require('path');
+const os = require('os');
+
+async function createFullAccessKey({ accountId, keyStore, networkId, nodeUrl }) {
+    const keyPair = KeyPair.fromRandom('ed25519');
+    const publicKey = keyPair.publicKey.toString();
+
+    const near = await connect({ keyStore, networkId, nodeUrl });
+    const account = await near.account(accountId);
+
+    await keyStore.setKey(networkId, publicKey, keyPair);
+    await account.addKey(publicKey);
+
+    return publicKey;
+}
+
+module.exports = {
+    createFullAccessKey,
+};
+
+if (require.main === module) {
```

Comment:
This ensures that the original logic w/ default parameters still works as expected when invoked via `node`. Most of the parameters are hardcoded but where the original implementation accepted command line arguments I've left the behavior the same.

Ideally I think each one would have a common CLI interface which would fall back on default arguments, allowing developers new to the ecosystem an opportunity to more seamlessly explore the API by tweaking parameters. However with all the changes in this PR it seems better left for a follow-up effort.